Code/GraphMol/MolDraw2D/MolDraw2DUtils.cpp

```diff
@@ -438,7 +438,18 @@ void drawMolACS1996(MolDraw2D &drawer, const ROMol &mol,
         << " and that may not look great with a pre-determined size."
         << std::endl;
   }
-  double meanBondLen = MolDraw2DUtils::meanBondLength(mol, confId);
+  double meanBondLen = 1.0;
+  if (drawer.drawOptions().prepareMolsBeforeDrawing &&
+      !mol.getNumConformers()) {
+    // compute 2D coordinates in a standard orientation.  This needs to be
+    // done on a copy because mol is const.
+    const bool canonOrient = true;
+    RWMol cpy(mol);
+    RDDepict::compute2DCoords(cpy, nullptr, canonOrient);
+    meanBondLen = MolDraw2DUtils::meanBondLength(cpy, confId);
+  } else {
+    meanBondLen = MolDraw2DUtils::meanBondLength(mol, confId);
```

Comment:
This is still going to throw in cases where `prepareMolsBeforeDrawing` is false and there is no conformer present

Reply:
But is that different from how drawMolecule() behaves in the same circumstances?  In this case, the user has explicitly said to use the molecule as it is supplied, but has supplied a molecule that isn't in a state to be drawn.  I think some form of shrug is not inappropriate at that point.

Reply:
Assuming we're talking about the same thing, I believe that it is different.
Here's the logic from `initDrawMolecule()` (https://github.com/rdkit/rdkit/blob/master/Code/GraphMol/MolDraw2D/DrawMol.cpp#L165) :
```
    if (drawOptions_.prepareMolsBeforeDrawing) {
      MolDraw2DUtils::prepareMolForDrawing(*drawMol_);
    } else if (!mol.getNumConformers()) {
      const bool canonOrient = true;
      RDDepict::compute2DCoords(*drawMol_, nullptr, canonOrient);
    }
```
So 2D coordinates will be generated even if `prepareMolsBeforeDrawing` is false.